docs/essentials/sensors.md

```diff
@@ -0,0 +1,540 @@
+
+The following code example demonstrates monitoring the accelerometer for changes:
+
+```csharp
+using Microsoft.Maui.Essentials;
+using System;
+
+namespace HelloMaui.Sensors
+{
+    public class AccelerometerTest
+    {
+        public void ToggleAccelerometer()
+        {
+            const SensorSpeed speed = SensorSpeed.UI;
+
+            try
+            {
+                if (Accelerometer.IsMonitoring)
+                {
+                    Accelerometer.Stop();
+                    Accelerometer.ReadingChanged -= Accelerometer_ReadingChanged;
+                }
+                else
+                {
+                    Accelerometer.ReadingChanged += Accelerometer_ReadingChanged;
+                    Accelerometer.Start(speed);
+                }
+            }
+            catch (FeatureNotSupportedException fnsEx)
+            {
+                // Feature not supported on device
+            }
+            catch (Exception ex)
+            {
+                // Other error has occurred.
+            }
+        }
+
+        void Accelerometer_ReadingChanged(object sender, AccelerometerChangedEventArgs e)
+        {
+            AccelerometerData data = e.Reading;
+
+            // Process Acceleration X, Y, Z
+            Console.WriteLine($"Reading: X: {data.Acceleration.X}, Y: {data.Acceleration.Y}, Z: {data.Acceleration.Z}");
+        }
+    }
+}
+```
+
+Accelerometer readings are reported back in **G**. A **G** is a unit of gravitation force equal to the gravity exerted by the earth's gravitational field $(9.81 m/s^2)$.
+
+The coordinate-system is defined relative to the screen of the device in its default orientation. The axes aren't swapped when the device's screen orientation changes.
+
+The **X** axis is horizontal and points to the right, the **Y** axis is vertical and points up and the **Z** axis points towards the outside of the front face of the screen. In this system, coordinates behind the screen have negative **Z** values.
+
+Examples:
+
+- When the device lies flat on a table and is pushed on its left side toward the right, the **X** acceleration value is positive.
+
+- When the device lies flat on a table, the acceleration value is +1.00 G or $(+9.81 m/s^2)$, which correspond to the acceleration of the device $(0 m/s^2)$ minus the force of gravity $(-9.81 m/s^2)$ and normalized as in G.
+
+<!-- TODO: Why are A and G mentioned here as if they're XYZ properties from the data? -->
+- When the device lies flat on a table and is pushed toward the sky with an acceleration of **A** $m/s^2$, the acceleration value is equal to $A+9.81$ which corresponds to the acceleration of the device $(+A m/s^2)$ minus the force of gravity $(-9.81 m/s^2)$ and normalized in **G**.
+
+### Platform-specific information (Accelerometer)
```

Comment:
Wondering if this needs calling out, seeing there isn't any platform-specific setup?